pkg/server/server.go

```diff
@@ -71,6 +72,11 @@ func New(c Config) *Server {
 		})
 	}
 
+	// Add the metrics endpoint in Prometehus format.
+	{
+		rtr.NewRoute().Methods("GET").Path("/metrics").Handler(promhttp.Handler())
+	}
```

Comment:
Here we expose the `/metrics` endpoint.

Reply:
this is similar to /metrics for alloy right?